en/99-apendice/2-repository-rules.md

```diff
@@ -0,0 +1,96 @@
+## Repository rules
+El principal material con el que trabajamos es código, con lo que se vuelve menester para la salud de todos los proyectos (y la nuestra) que mantengamos una serie de reglas para trabajar ordenadamente en los repositorios.
```

Comment:
The main resource we work with is code, so it's vital to the health of each project (and ours) that we keep a set of rules that'll keep us working in order.